rmgpy/molecule/moleculeTest.py

```diff
@@ -1276,6 +1278,123 @@ def testSaturateAromaticRadical(self):
+16 C u0 p0 c0 {6,B} {17,B} {32,S}
+17 C u0 p0 c0 {9,B} {16,B} {21,S}
+18 C u0 p0 c0 {10,B} {11,B} {24,S}
+19 C u0 p0 c0 {12,B} {13,B} {27,S}
+20 C u0 p0 c0 {14,B} {15,B} {30,S}
+21 H u0 p0 c0 {17,S}
+22 H u0 p0 c0 {9,S}
+23 H u0 p0 c0 {10,S}
+24 H u0 p0 c0 {18,S}
+25 H u0 p0 c0 {11,S}
+26 H u0 p0 c0 {12,S}
+27 H u0 p0 c0 {19,S}
+28 H u0 p0 c0 {13,S}
+29 H u0 p0 c0 {14,S}
+30 H u0 p0 c0 {20,S}
+31 H u0 p0 c0 {15,S}
+32 H u0 p0 c0 {16,S}
+""")
+        perylene2 = Molecule().fromSMILES('c1cc2cccc3c4cccc5cccc(c(c1)c23)c54')
+        for isomer in generateAromaticResonanceIsomers(perylene2):
+            if perylene.isIsomorphic(isomer):
+                break
+        else:  # didn't break
+            self.fail("{} isn't isomorphic with any aromatic forms of {}".format(
+                            perylene.toSMILES(),
+                            perylene2.toSMILES()
+                        ))
+
+    def testFusedAromatic2(self):
+        """Test we can make aromatic naphthalene from both adjlist and SMILES"""
+        naphthalene = Molecule().fromAdjacencyList("""
+1  C u0 p0 c0 {2,B} {3,B} {4,B}
+2  C u0 p0 c0 {1,B} {5,B} {6,B}
+3  C u0 p0 c0 {1,B} {8,B} {13,S}
+4  C u0 p0 c0 {1,B} {9,B} {14,S}
+5  C u0 p0 c0 {2,B} {10,B} {17,S}
+6  C u0 p0 c0 {2,B} {7,B} {18,S}
+7  C u0 p0 c0 {6,B} {8,B} {11,S}
+8  C u0 p0 c0 {3,B} {7,B} {12,S}
+9  C u0 p0 c0 {4,B} {10,B} {15,S}
+10 C u0 p0 c0 {5,B} {9,B} {16,S}
+11 H u0 p0 c0 {7,S}
+12 H u0 p0 c0 {8,S}
+13 H u0 p0 c0 {3,S}
+14 H u0 p0 c0 {4,S}
+15 H u0 p0 c0 {9,S}
+16 H u0 p0 c0 {10,S}
+17 H u0 p0 c0 {5,S}
+18 H u0 p0 c0 {6,S}
+""")
+        naphthalene2 = Molecule().fromSMILES('C1=CC=C2C=CC=CC2=C1')
+        for isomer in generateAromaticResonanceIsomers(naphthalene2):
+            if naphthalene.isIsomorphic(isomer):
+                break
+        else:  # didn't break
+            self.fail("{} isn't isomorphic with any aromatic forms of {}".format(
+                            naphthalene.toSMILES(),
+                            naphthalene2.toSMILES()
+                        ))
+
+    def testFusedAromatic3(self):
+        """Test we can make aromatic pyrene_rad from both adjlist and SMILES"""
+        pyrene_rad = Molecule().fromAdjacencyList("""
+multiplicity 2
+1  C u0 p0 c0 {2,B} {3,B} {5,B}
+2  C u0 p0 c0 {1,B} {4,B} {6,S}
+3  C u0 p0 c0 {1,B} {8,B} {9,B}
+4  C u0 p0 c0 {2,B} {10,B} {11,S}
+5  C u0 p0 c0 {1,B} {7,B} {15,S}
+6  C u0 p0 c0 {2,S} {12,S} {16,D}
+7  C u0 p0 c0 {5,B} {13,B} {17,S}
+8  C u0 p0 c0 {3,B} {13,B} {19,S}
+9  C u0 p0 c0 {3,B} {10,B} {20,S}
+10 C u0 p0 c0 {4,B} {9,B} {21,S}
+11 C u1 p0 c0 {4,S} {14,S} {22,S}
+12 C u0 p0 c0 {6,S} {14,D} {24,S}
+13 C u0 p0 c0 {7,B} {8,B} {18,S}
+14 C u0 p0 c0 {11,S} {12,D} {23,S}
+15 C u0 p0 c0 {5,S} {16,D} {25,S}
+16 C u0 p0 c0 {6,D} {15,D}
+17 H u0 p0 c0 {7,S}
+18 H u0 p0 c0 {13,S}
+19 H u0 p0 c0 {8,S}
+20 H u0 p0 c0 {9,S}
+21 H u0 p0 c0 {10,S}
+22 H u0 p0 c0 {11,S}
+23 H u0 p0 c0 {14,S}
+24 H u0 p0 c0 {12,S}
+25 H u0 p0 c0 {15,S}
```

Comment:
Haha oops. I wasted time photoshopping out one of the hydrogens, without realizing I'd successfully removed the one opposite it before generating the image (I thought my attempt had failed). Try this one:
<img width="294" alt="screen shot 2016-09-14 at 11 01 48 pm" src="https://cloud.githubusercontent.com/assets/93807/18536908/45c1c376-7acf-11e6-94f2-82b95d675fce.png">

It may not be a sensible resonance form; I meant "valid" for the way RMG currently handles delocalization and Kekulization. 

This may be naïve, but my quick thinking (I think based on what RMG attempts)  is any bonding arrangement which allows you to put all the atoms (including H's) in the right place, and has the same overall number of radicals (hence multiplicity), is a resonance structure.